tests/unit-tests/test_dataloading.py

```diff
@@ -490,9 +493,15 @@ def test_GSgnnData_edge_feat():
         for _, _, blocks in dataloader:
             edge_feat_list = gdata.get_blocks_edge_feats(blocks, efeat_fields)
             assert len(edge_feat_list) == 2
-            assert edge_feat_list[0][('n0', 'r0', 'n1')].shape[1] == 2
+            if ('n0', 'r0', 'n1') in edge_feat_list[0]:
```

Comment:
Why there is an if here?

Reply:
Because the test dummy data are randomly generated, and blocks are randomly sampled, the sampled subgraph cannot make sure features always existing for some edge types. Therefore, need to check if the edge type exist first.